src/loadouts.py

```diff
@@ -0,0 +1,104 @@
+# This module defines the Settings class which provides a singleton instance to manage application settings,
+# allowing settings to be loaded from a file, updated dynamically, and used across the application.
+# The Settings class follows a singleton pattern.
+
+import constants
+import json
+import utilities
+
+class Loadout:
+    """
+    Simple data class designed to handle the storage and management of different 
+    configurations of macro keys for a particular setup.
+    This class is used primarily in the settings.
+    """
+    def __init__(self, name, macroKeys):
+        self.name = name
+        self.macroKeys = macroKeys
+
+class LoadoutManager:
+    def __init__(self):
+        self._observers = []
+        self.loadouts = {utilities.generateUuid(): Loadout("Loadout 1", {"1": "1"})}
```

Comment:
We should take this opportunity to design some more meaningful default loadouts
Maybe just 2 loadouts: Bugs, Automatons with ~5 stratagems each

Reply:
This has been addressed